patterns.md

```diff
@@ -47,8 +48,11 @@ class Teams(generic.ListView):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        # Serialise the queryset and remove object_list from the ctx.
+
+        # Serialise the queryset and remove object_list from the ctx so the 
```

Comment:
Brexit spelling of 'serialise'. I think we decided to go for US English. Presumably comments should be consistent. 🤷‍♂️